test/e2e/network/loadbalancer.go

```diff
@@ -1624,3 +1650,140 @@ var _ = common.SIGDescribe("LoadBalancers ESIPP [Slow]", func() {
 		}
 	})
 })
+
+func testRollingUpdateLBConnectivityDisruption(f *framework.Framework, externalTrafficPolicy v1.ServiceExternalTrafficPolicyType, minSuccessRate float64) {
+	cs := f.ClientSet
+	ns := f.Namespace.Name
+	name := "test-lb-rolling-update"
+	labels := map[string]string{"name": name}
+	gracePeriod := int64(10)
```

Comment:
I recommend using a much higher value here, like 30. Even higher like 60 might be better, although that might  make the test significantly slower.  

Reason is that this value has to be at least ("LB health check interval" x "Unhealthy threshold") + some buffer space. The value ends up depending on the LB configuration for the healthcheck, so something like 30s or 60s  would likely work across most default health check intervals.

Reply:
If we increase the grace period to something like 60s, we might want to tweak the RollingUpdate strategy to use a higher `maxUnavailable` and `maxSurge` value, like 10%. This way the test time is still reasonable when testing against really large clusters

Reply:
cc @wojtek-t re: large scale scalability test -- I think 60s termination grace period is fine with maxUnavailable: 10% but I could be missing something

Reply:
Didn't look into PR, so commenting only based on the comment thread.

Reply:
Sorry - clicked send too quickly...

60s grace period with 10% unavailable, means rolling update takes 10m+ - it's somewhat large, but assuming we don't have multiple such upgrades, it seems acceptable